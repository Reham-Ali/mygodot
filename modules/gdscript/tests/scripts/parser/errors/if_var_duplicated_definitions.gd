func test():
    if var x := 1, var x := 2:
        print("t")
