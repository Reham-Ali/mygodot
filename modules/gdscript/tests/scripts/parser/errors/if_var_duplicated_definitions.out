GDTEST_PARSER_ERROR
There is already a variable named "x" declared in this scope.
